go/vt/vttablet/tabletserver/dt_executor_test.go

```diff
@@ -419,6 +421,62 @@ func TestExecutorSetRollback(t *testing.T) {
 	require.Contains(t, err.Error(), "could not transition to ROLLBACK: aa")
 }
 
+func TestExecutorUnresolvedTransactions(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	txe, tsv, db := newTestTxExecutor(t, ctx)
```

Comment:
nitpick: we could return a single closer here. everywhere we are using `newTestTxExecutor`, we then have to add two closers